documentation/modules/oauth/proc-oauth-broker-config.adoc

```diff
@@ -7,19 +7,22 @@
 
 This procedure describes how to configure Kafka brokers so that the broker listeners are enabled to use {oauth} authentication using an authorization server.
 
+{oauth} requires configuration of TLS listeners.
+Plain listeners are not recommended.
+
 If the authorization server is using certificates signed by the trusted CA and matching the {oauth} server hostname, TLS connection works using the default settings.
-Otherwise, you have two connection options for your external listener configuration when delegating token validation to the authorization server:
+Otherwise, you have two connection options for your listener configuration when delegating token validation to the authorization server:
 
 . TLS connection to the authorization server with trusted certificates
 . Direct connection using an introspection endpoint configuration
```

Comment:
This sounds wrong. The TLS will need to be adresses in this case as well.

Reply:
Maybe it makes more sense with the TLS:

Otherwise, you have two connection options for your listener configuration when delegating token validation to the authorization server:

. Connection to the authorization server with trusted certificates
. Direct connection using an introspection endpoint configuration

Reply:
Maybe _Using the trusted signing certificates obtained from the to the authorization server_? The "trusted certificates make it confusing with TLS - at least for me.